ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -296,7 +296,6 @@ const VulnMgmtCves = ({
     const [selectedCveIds, setSelectedCveIds] = useState([]);
     const [bulkActionCveIds, setBulkActionCveIds] = useState([]);
     const { isFeatureFlagEnabled } = useFeatureFlags();
-    const showVMUpdates = isFeatureFlagEnabled('ROX_FRONTEND_VM_UPDATES');
     const usesPostgres = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
```

Comment:
I'd bring this into alignment with the pattern everywhere else.

```suggestion
    const showVMUpdates = isFeatureFlagEnabled('ROX_POSTGRES_DATASTORE');
```

And likewise just use `idAttribute={showVMUpdates ? 'id' : 'cve'}` in the change below.